omod/src/test/java/org/openmrs/module/fhir2/providers/r3/ConditionFhirR3ResourceProviderWebTest.java

```diff
@@ -438,4 +510,25 @@ private void verifyURI(String uri) throws Exception {
 		assertThat(response.getContentType(), equalTo(FhirMediaTypes.JSON.toString()));
 		assertThat(readBundleResponse(response).getEntry().size(), greaterThanOrEqualTo(1));
 	}
+	
+	@Test
+	public void deleteCondition_shouldDeleteCondition() throws Exception {
+		when(conditionService.delete(CONDITION_UUID)).thenReturn(condition);
```

Comment:
The error for this test seems to indicate that `conditionService.delete()` id not being called or is not being called with the right argument. This should probably be stepped through in a debugger to figure out why it's failing.

Reply:
Thanks alot @ibacher , think this PR  is so disorganised however i tried commenting out lines seeing which line is causing errors but soon i will make everything in order, thanks for your comments, honestly am not sure why test failure but still, Am not it.